conf/base.config

```diff
@@ -74,11 +74,11 @@ process {
         label           = "process_high"
         memory          = { check_max( 36.GB * task.attempt, 'memory' ) }
     }
-    withName: 'GATK4_MARKDUPLICATES|GATK4_MARKDUPLICATESSPARK' {
+    withName: 'GATK4_MARKDUPLICATES|GATK4SPARK_MARKDUPLICATES' {
```

Comment:
can you add the name changes to the changelog, I think we even had a table for that. Relevant for @ameynert and others for their institution specific configs

Reply:
Done